### ISSUE TITLE
don't export revolved surfaces in IGES

### ISSUE THREAD

**Issue by pafurijaz** [gcad3d/gcad3d#41]
Hi I'm learning to use gCAD3D a little but it does not export revolved surface and the extrusions of the planes is wrong, but the display is correct in gCAD3D

this is my code
```
# 2021/12/30-17:29:56
# Precison input
P20=0,0,0
P21=250,0,0
P22=500,0,0
P23=250,0,250
P24=250,1000,350
S20=BSP P(0 0 250) P(100 570 280) P(250 1000 350)
S21=BSP P(500 0 250) P(250 1000 350)
C20=P(250 0 250) 250 D(RY)
L20=P(250 1000 350) P(250 1000 850)
L22=P(250 0 250) P(250 250 250)
R(RY P24)
C21=ARC P(P24) ANG(0) ANG(-180) 50 CW
L21=P(0 150 0) P(100 150 0)
R(RZ)
A20=SRV R(L21) C21 180 360 0 1 CW
A21=SRU C20 D(L22)
A22=SRU L20 D(L21)
ATTS "C001155" A20
ATTS "C0044FF" A21
ATTS "C990000" A22
```
![image](https://user-images.githubusercontent.com/11751383/147782130-578e172e-ed52-4b21-a741-d6cccf7e1e91.png)
this is the IGES imported in FreeCAD
![image](https://user-images.githubusercontent.com/11751383/147782282-d6b31cbd-7589-4244-8aca-b21a1d0c668a.png)
and this the imported IGES in gCAD3D
![image](https://user-images.githubusercontent.com/11751383/147782418-78018ca4-2c6b-4e89-a322-cd5d5b180f44.png)

**Comment by ckitgui** [comment]
Bug Iges-export-planar-surf can be reproduced; confirmed;
Bug Iges-export-toroid-surf can be reproduced; confirmed;
export of the cylindric-surf is OK.
Can be verified using cad_assistant_1.6.0_2021-10-05_lin64.appimage.
These bugs will be fixed with next version of gcad3d.
Franz

**Comment by pafurijaz** [comment]
Thanks for support, I have also some issues with STEP export I don't make an other report if it is not necessary but i report here in this one.

The gCAD3D is Below and also the of imported STEP result of screen shots in various applications.

```
# 2021/12/31-11:12:52
# Solid and surfaces

B20=CON P(0 1000 0) P(0 1000 500) VAL(250) VAL(25)
B21=SPH P(500 0 250) 250
B22=TOR P(1100 0 100) D(RZ) VAL(250) VAL(100)
R20=PERP P(0 500 0) D(RZ) D(RX) 50
L20=P(500 500 0) P(500 500 1000) UNL
A20=SPH RZ 250 60 360 0 180
L21=P(500 1000 0) P(500 1000 500)
A21=CYL R(L21) 250 75 360 500 1
S20=BSP P(1289 930 0) P(1277 1423 0) P(958 1965 0) P(1200 2562 0)
S21=BSP P(1929 949 250) P(1938 1324 250) P(1840 1835 250) P(2156 2573 250)
S22=BSP P(2651 898 0) P(2451 1513 0) P(2586 1916 0) P(2723 2576 0)
A22=BSP U(S20 S21 S22)  VAL(2)

```
![image](https://user-images.githubusercontent.com/11751383/147821053-1131b5e6-fd6b-435b-a292-c3b75dcb3947.png)

CAD ASSISTANT
![image](https://user-images.githubusercontent.com/11751383/147821127-686f52f4-28c2-4841-93cb-b5207b69f496.png)

FreeCAD
![image](https://user-images.githubusercontent.com/11751383/147821154-7f20e8a4-be5c-4644-918d-6a96e0c8f798.png)

Moi3D
![image](https://user-images.githubusercontent.com/11751383/147821221-5df09b53-07f8-46c0-9383-4823cb604823.png)

gCAD3D imported STEP
![image](https://user-images.githubusercontent.com/11751383/147821350-c040cbe9-2ce2-41b2-b852-1400de2017e3.png)

This file has also doesn't works if you export it into IGES the only surfaces well exported is the loft one

Below other test of exporting capability
this is the OBJ exported
![image](https://user-images.githubusercontent.com/11751383/147821612-86c8de7c-10a7-401f-9b34-c9b77f90e9f2.png)

Also the normal of OBJ and WRML1 are flipped anomaly in the same surfaces and also lofted surfaces has an extra surface on bottom
![image](https://user-images.githubusercontent.com/11751383/147821797-f65e7e9c-0d29-4426-99ca-6def708f1e6d.png)

VRML1 imported in FreeCAD with faces that is filliped
![image](https://user-images.githubusercontent.com/11751383/147822016-b739c563-99da-4ea9-8047-b55173fb8a6c.png)

This is the VRML2 file imported in FreeCAD
![image](https://user-images.githubusercontent.com/11751383/147822067-88229ac0-73c4-48fa-9961-2d409404d3de.png)

With STL export works fine but **with DXF export gCAD3D crash**

Thanks

**Comment by gcad3d** [comment]
ad source example from Dez 30.2021:
R(RY P24) # activate 2D-plane OK
..
R(RZ)     # activate 2D with basic plane; NOT OK; reset to 3D with RZ only !
RZ        # reset to 3D      <<<<

- export iges is beeing rewritten completly; partially done with last version.

**Comment by gcad3d** [comment]
Fixed with completly new ige-export V-2.60.04.
